tests/components/layout/test_cond.py

```diff
@@ -86,3 +88,21 @@ def test_cond_no_else():
     # Props do not support the use of cond without else
     with pytest.raises(ValueError):
         cond(True, "hello")
+
+
+def test_cond_foreach(mocker: pytest_mock.MockFixture):
+    var_name = "abc"
+    mocker.patch("pynecone.utils.get_unique_variable_name", return_value=var_name)
+
+    cond_component = Cond.create(
+        True,  # pyright: reportGeneralTypeIssues=false
+        Foreach.create(
+            ["Tommy", "is", "cool"],  # pyright: reportGeneralTypeIssues=false
```

Comment:
Suggested we only see pyright error as a warning until it's fixed.

The pyright error seems off with our type hints:
```
Argument of type "list[atr]" cannot be assigned to parameter "iterable" of type "Var" in function "create"
```